src/Popover.js

```diff
@@ -48,7 +50,7 @@ const Popover = React.createClass({
     };
 
     return (
-      <div {...this.props} className={classNames(this.props.className, classes)} style={style} title={null}>
+      <div role='tooltip' {...this.props} className={classNames(this.props.className, classes)} style={style} title={null}>
```

Comment:
Not in this case. There are a preset number of valid roles that that are allowed her according to the HTML/ARIA spec, out of those "tooltip" best describes the role of a popover.